Ginger/Ginger/ValueExpression/ValueExpressionEditorPage.xaml.cs

```diff
@@ -463,40 +565,42 @@ private void AddVBSIfEval(TreeViewItem tviVars, string Desc, string Eval)
 
         private void AddEnvParams()
         {
+            TreeViewItem Parent = AddOrGetCategory("Data");
+            //TreeViewItem child = AddOrGetSubCategory("Environments", Parent);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6356765)